website/docs/troubleshooting/troubleshooting-podman.md

```diff
@@ -233,3 +233,26 @@ This might appear when either:
 3. Restart the Podman machine to recreate and activate the default Docker socket path.
 
 _Note:_ If Docker Desktop is started again, it will automatically re-alias the default Docker socket location and the Podman compatibility warning will re-appear.
+
+## Uninstalling Podman Desktop preserves the old configuration data  
+
+#### Issue
+
+After uninstalling Podman Desktop, the configuration data persists even though it is not needed for a fresh installation.
+
+
+#### Solution
+
+**_Deleting Podman Desktop configuration_**
+
+- Perform one of the following steps based on your operating system:
+   - On macOS, go to the _~/.local/share/containers/podman-desktop/_ directory and delete the _configuration_ folder.
```

Comment:
```suggestion
   - On macOS, go to the _~/.local/share/containers/podman-desktop/_ directory and delete the _configuration_ folder.
```

we should use code formatting to indicate directory names like other markdown docs, example:

`~/.local/share/containers/podman-desktop/`

Reply:
+1 to that. 

Am I missing something here? Aren't these two directory locations the same? Why not put all 3 on the same line?

Reply:
Thanks for catching this. Yes, the directory location is same on all supported OSs.